style/team.qmd

```diff
@@ -0,0 +1,51 @@
+# Team work
+
+## Weekly stand up
+
+Every Monday from 10:00 - 11:00 we have a team meeting. It starts with everyone answering the following questions:
+
+- ⭐ What went well last week
+- 🦉 What lessons did you learn
+- 🎯 Goal(s) for this week
+
+If you are going to be absent on a Monday morning, you might want to share your three-things on Slack in advance.
+
+We then discuss any items that team members have added to the [rolling agenda](https://docs.google.com/document/d/1WukwitMpDU9RexZaXHIvEf-vV6iGKLjCsdAVfKcIMPE/).
+
+## Line management
+
+Your line manager should arrange (1 : 1) meetings with you once a fortnight.
+
+## Communication
+
+We use MS Teams to communicate about project related work. 
+We have a private slack channel on NHS-R slack which we use for general team chat and more casually. (Slack has much better emojis 🤸🎮🦥)
+
+## Data Science Trophy 
+
+One of the ways we celebrate each other and the successes we have is by presenting each other with a “trophy” 🏆 in the slack channel. This is for good code, problem solving, helping someone anything that deserves a special mention.
+
+## Working Times 
+
+Core hours vary considerably between team members and there is a lot of flexibility. Team members may send messages out of hours, but that is purely because it works for them, and will not expect anything immediate in response. Flexibility is not only allowed but recommended if it benefits you. 
```

Comment:
```suggestion
Core hours vary considerably between team members and there is a lot of flexibility. Team members may send messages out of hours, but that is purely because it works for them, and will not expect anything immediate in response. Flexibility is not only allowed but recommended if it benefits you. Please agree core hours with your line manager and where possible communicate any changes in your core hours to all team members via Slack (e.g. "I'm out all morning because my car is broken- working later on").
```

Reply:
I edited to suggest using out-of-office in calendar as well for communicating changes to core hours.